Getting the following error when testing your branch locally:
```C#
C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Text.Json\tests\System.Text.Json.Tests\System.Text.Json.Tests.csproj : error NU1202: Package xunit.runner.visualstudio 2.4.4 is not compatible with net462 (.NETFramework,Version=v4.6.2). Package xunit.runner.visualstudio 2.4.4 supports: [C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Text.Json\System.Text.Json.sln]
C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Text.Json\tests\System.Text.Json.Tests\System.Text.Json.Tests.csproj : error NU1202:   - net472 (.NETFramework,Version=v4.7.2) [C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Text.Json\System.Text.Json.sln]
C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Text.Json\tests\System.Text.Json.Tests\System.Text.Json.Tests.csproj : error NU1202:   - netcoreapp3.1 (.NETCoreApp,Version=v3.1) [C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Text.Json\System.Text.Json.sln]
C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Text.Json\tests\System.Text.Json.Tests\System.Text.Json.Tests.csproj : error NU1202:   - uap10.0.16299 (UAP,Version=v10.0.16299) [C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Text.Json\System.Text.Json.sln]
```
Would this change require increasing `NetFrameworkMinimum` to `net472` perhaps?

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -155,7 +155,7 @@
     <MicrosoftDotNetHotReloadUtilsGeneratorBuildToolVersion>1.0.2-alpha.0.22178.2</MicrosoftDotNetHotReloadUtilsGeneratorBuildToolVersion>
     <XUnitVersion>2.4.2-pre.22</XUnitVersion>
     <XUnitAnalyzersVersion>0.12.0-pre.20</XUnitAnalyzersVersion>
-    <XUnitRunnerVisualStudioVersion>2.4.4</XUnitRunnerVisualStudioVersion>
+    <XUnitRunnerVisualStudioVersion>2.4.5</XUnitRunnerVisualStudioVersion>
     <CoverletCollectorVersion>3.1.2</CoverletCollectorVersion>
     <NewtonsoftJsonVersion>12.0.3</NewtonsoftJsonVersion>
     <SQLitePCLRawbundle_greenVersion>2.0.4</SQLitePCLRawbundle_greenVersion>

